scripts/appveyor-tool.ps1

```diff
@@ -129,6 +129,7 @@ Function InstallRtools {
     $gcc_path = $env:GCC_PATH
   }
   $env:PATH = $RtoolsDrive + '\Rtools\bin;' + $RtoolsDrive + '\Rtools\MinGW\bin;' + $RtoolsDrive + '\Rtools\' + $gcc_path + '\bin;' + $env:PATH
+  $env:BINPREF=$RtoolsDrive + '/Rtools/mingw_$(WIN)/bin/'
```

Comment:
Thanks! I'll wait for the tests to finish, merge, and do some more testing after merging. Perhaps we don't need to put gcc on the PATH at all, just set it in BINPREF.